src/main/java/com/auctionapp/api/model/dto/UserDto.java

```diff
@@ -3,14 +3,20 @@
 import java.sql.Timestamp;
 import java.util.UUID;
 
+import com.auctionapp.api.model.entities.UserRole;
+
 public class UserDto {
-	private final UUID id;
-	private final String firstName;
-	private final String lastName;
-	private final String email;
-	private final String password;
-	private final Timestamp createdAt;
-	private final Timestamp updatedAt;
+	private UUID id;
+	private String firstName;
+	private String lastName;
+	private String email;
+	private String password;
+	private Timestamp createdAt;
+	private Timestamp updatedAt;
+	private UserRole role;
```

Comment:
Why did you remove `final` here?

Reply:
I honestly don't remember purposefully deleting it, it might be that I did it while trying to fix the bug I had with registration and login, so I was checking everything that was different from the last working commit. I put back final now.